capstone/src/main/webapp/authenticate.js

```diff
@@ -18,11 +18,11 @@ function createDynamicSigninButton() {
 }
 
 function createLogoutButton(){
-    const logoutButtonElement = document.createElement('button');
-    logoutButtonElement.id = 'signout_button';
-    logoutButtonElement.style.display = "block";
-    logoutButtonElement.innerText = "Sign Out";
-    return logoutButtonElement;
+    let domparser = new DOMParser();
+    let doc = domparser.parseFromString(`
```

Comment:
Nit: Recommend using `const` when you don't re-assign a variable.

Reply:
got it